callisto/delivery/forms.py

```diff
@@ -97,72 +97,109 @@ def clean_key(self):
-                                    max_length=50,
-                                    widget=forms.TextInput(attrs={'placeholder': 'ex. (555) 555-5555'}),)
-
-    voicemail = forms.CharField(label="Is it ok to leave a voicemail? If so, what would you like the message to refer to?",
-                                    required=False,
-                                    max_length=256,
-                                    widget=forms.TextInput(attrs={'placeholder': "ex. Yes, please just say you're following up from Callisto."}),)
+                                   required=True,
+                                   max_length=50,
+                                   widget=forms.TextInput(attrs={'placeholder': 'ex. (555) 555-5555'}),)
+
+    voicemail = forms.CharField(
+        label="Is it ok to leave a voicemail? If so, what would you like the message to refer to?",
+        required=False,
+        max_length=256,
+        widget=forms.TextInput(
+            attrs={
+                'placeholder': "ex. Yes, please just say you're following up from Callisto."}),
+        )
 
     email = forms.EmailField(label="If you can't be reached by phone, what's the best email address to reach you?",
-                                    required=True,
-                                    max_length=256,
-                                    widget=forms.TextInput(attrs={'placeholder': 'ex. myname@gmail.com'}),)
-
-    contact_notes = forms.CharField(label="Any notes on what time of day is best to reach you? A {0} staff member will "
-                                          "try their best to accommodate your needs.".format(settings.SCHOOL_SHORTNAME),
-                                    required=False,
-                                    widget=forms.Textarea(attrs={'placeholder': "ex. I have class from 9-2 most weekdays and work in the evenings, so mid afternoon is best (around 2:30-5pm)."
-                                                                                "Wednesdays and Thursdays I have class the entire day, so those days don't work at all.",
-                                                                 'max_length': 5000}),)
-
-    email_confirmation = forms.ChoiceField(choices=[(True, "Yes"), (False, "No, thanks")],
-                                           label="Would you like us to send you a confirmation email with information about your rights in the reporting process, and where to get support and find resources on campus?",
-                                           required=True,
-                                           widget=forms.RadioSelect)
+                             required=True,
+                             max_length=256,
+                             widget=forms.TextInput(attrs={'placeholder': 'ex. myname@gmail.com'}),)
+
+    contact_notes = forms.CharField(
+        label=("Any notes on what time of day is best to reach you? A {school} staff member will"
```

Comment:
@kelseyq you okay with this way of breaking up long strings? I'm using python's implied line continuation within parentheses, brackets and braces. PEP8 [says this is the preferred method](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).